### PR TITLE
M5942  As BBMRI-ERIC I want to show logo's for standards at the biobank level as well

### PR DESCRIPTION
**Pull request by marikaris**
Model + public data: 
[eric_data_quality.xlsx](https://github.com/molgenis/molgenis-app-biobank-explorer/files/2309503/eric_data_quality.xlsx)

Additional dummy data with quality data for Graz biobank:
[eu_bbmri_eric_col_qual_info_2018-08-22_10_47_41.xlsx](https://github.com/molgenis/molgenis-app-biobank-explorer/files/2309523/eu_bbmri_eric_col_qual_info_2018-08-22_10_47_41.xlsx)
[eu_bbmri_eric_bio_qual_info_2018-08-22_10_47_26.xlsx](https://github.com/molgenis/molgenis-app-biobank-explorer/files/2309524/eu_bbmri_eric_bio_qual_info_2018-08-22_10_47_26.xlsx)

Add some more logo's + quality parameters to the quality tables.

#### Checklist
- [x] Functionality works & meets specifications
- [x] Code reviewed
- [x] Code unit/integration/system tested
- [x] User documentation updated
- [x] Clean commits
- [x] No warnings during install
- [x] Updated flow typing

### CODE REVIEW

File: src/components/cards/BiobankReportCard.vue [hunk 1]
```diff
@@ -4,38 +4,65 @@
...
+              </div>
+            </div>
           </div>
-          <div class="col-md-4">
-            <ul class="list-unstyled">
-              <li><b>{{ biobank.data.contact.first_name }} {{ biobank.data.contact.last_name }}</b></li>
-              <li>Email: {{ biobank.data.contact.email }}</li>
-              <li>Tel: {{ biobank.data.contact.phone }}</li>
-            </ul>
+          <div class="col-2">
```
Comment (fdlk) [inline src/components/cards/BiobankReportCard.vue]: The bigger this card gets, the more attractive it becomes to split off some subcomponents

File: src/components/cards/BiobankReportCard.vue [hunk 2]
```diff
@@ -119,13 +166,17 @@
     name: 'biobank-report-card',
     data () {
       return {
-        collapsed: true
+        collapsed: true,
+        quality_logo: {height: 4, width: 9}
```
Comment (fdlk) [inline src/components/cards/BiobankReportCard.vue]: If these values are fixed anyways, I'd put it in the style section

File: src/components/cards/BiobankReportCard.vue [hunk 3]
```diff
@@ -67,6 +94,22 @@
                   <a :href="biobank.data[attribute.name]" target="_blank">{{ biobank.data[attribute.name] }}</a>
                 </span>
 
+                <span v-else-if="attribute.name === 'quality'">
+                  <table class="table table-sm">
+                    <tr v-for="quality in biobank.data[attribute.name]">
```
Comment (fdlk) [inline src/components/cards/BiobankReportCard.vue]: -> `biobank.data.quality` is probably a bit clearer

File: src/components/cards/BiobankReportCard.vue (hunk 2 above)
Comment (marikaris) [reply src/components/cards/BiobankReportCard.vue]: I did this, but the styling was somehow not applied when I did.

**Pull request merged**